packages/docusaurus-plugin-pwa/package.json

```diff
@@ -20,17 +20,17 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@babel/core": "^7.19.0",
-    "@babel/preset-env": "^7.19.0",
+    "@babel/core": "^7.19.6",
+    "@babel/preset-env": "^7.19.4",
     "@docusaurus/core": "^3.0.0-alpha.0",
     "@docusaurus/theme-common": "^3.0.0-alpha.0",
     "@docusaurus/theme-translations": "^3.0.0-alpha.0",
     "@docusaurus/types": "^3.0.0-alpha.0",
     "@docusaurus/utils": "^3.0.0-alpha.0",
     "@docusaurus/utils-validation": "^3.0.0-alpha.0",
-    "babel-loader": "^8.2.5",
+    "babel-loader": "^9.0.0",
```

Comment:
major deps upgrades often have breaking changes that might lead ourselves to have breaking changes too. 

https://github.com/babel/babel-loader/releases/tag/v9.0.0

In this case it doesn't seem to be a problem (even though we allow configuring babel plugins) but the 9.0 was released only 20h ago. 

For safety, I'd prefer to stay on 8.x, and keep such upgrades for the next major versions.

IE: don't do any risky change just before a minor release. If a PR feels too risky I'd prefer to not backport it. Releasing a minor takes more time than before and I don't want to have to release a 2.2.1 if possible.

In general, I'd prefer to avoid having to backport a deps upgrade PR just before a release. Let's merge those PRs a bit upfront, see if there's any issue, and only then backport it.